docs/manual/ru/introduction/Installation.html

```diff
@@ -0,0 +1,201 @@
+		импортом"). Другие популярные части библиотеки, такие как элементы управления, загрузчики и эффекты
+		постобработки,
+		должны быть импортированы из подпапки [link:https://github.com/mrdoob/three.js/tree/dev/examples/jsm
+		examples/jsm].
+		Чтобы узнать больше, смотрите <em>Примеры</em> ниже.
+	</p>
+
+	<p>
+		Узнайте больше о модулях npm по ссылке [link:https://eloquentjavascript.net/20_node.html#h_J6hW/SmL/a Eloquent
+		JavaScript: Installing with npm](Выразительный JavaScript: Установка с помощью npm).
+	</p>
+
+	<h2>Установка с CDN или статического хостинга</h2>
+
+	<p>
+		В three.js библиотеку можно использовать без какой-либо системы сборки, либо загрузив файлы на свой собственный
+		веб-сервер, либо используя существующий CDN. Поскольку библиотека полагается на модули ES, любой скрипт, который
+		ссылается на нее, должен использовать <em>type="module"</em>, как показано ниже.
+		Также требуется определить схему импорта, которая разрешает пустой модуль, указанный как `three`.
+	</p>
+
+	<code>
+		&lt;script async src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js">&lt;/script>
+
+		&lt;script type="importmap">
+		{
+		"imports": {
+		"three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js"
+		}
+		}
+		&lt;/script>
+
+		&lt;script type="module">
+
+		import * as THREE from 'three';
+
+		const scene = new THREE.Scene();
+
+		&lt;/script>
+	</code>
+
+	<p>
+		Поскольку импорт карт еще не поддерживается всеми браузерами, необходимо добавить полифил *es-module-shims.js*.
+	</p>
+
+	<h2>Примеры</h2>
+
+	<p>
+		Ядро three.js сосредоточен на наиболее важных компонентах 3D-движка. Многие другие полезные компоненты, такие
+		как
+		элементы управления, загрузчики и эффекты постобработки, являются частью каталога
+		[link:https://github.com/mrdoob/three.js/tree/dev/examples/jsm examples/jsm]. Они называются
+		"примерами(examples)",
+		потому что, хотя вы можете использовать их в готовом виде, они также предназначены для ремиксов и кастомизации.
+		Эти
+		компоненты всегда синхронизируются с основной библиотекой, в то время как аналогичные пакеты сторонних
+		разработчиков
+		в npm поддерживаются разными пользователями и могут быть устаревшими.
+	</p>
+
+	<p>
+		Примеры не нужно <em>устанавливать</em> отдельно, но их нужно <em>импортировать</em> отдельно. Если three.js был
+		установлен с помощью npm, вы можете загрузить компонент [page:OrbitControls](Управление орбитой) с помощью:
+	</p>
+
+
+	<code>
+		import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+
+		const controls = new OrbitControls( camera, renderer.domElement );
+	</code>
+
+	<p>
+		Если three.js был установлен с CDN, используйте тот же CDN для установки других компонентов:
+	</p>
+
+	<code>
+		&lt;script type="module">
+
+		import { OrbitControls } from 'https://unpkg.com/three@&lt;version&gt;/examples/jsm/controls/OrbitControls.js';
+
+		const controls = new OrbitControls( camera, renderer.domElement );
```

Comment:
One thing: This section has recently changed. An import map was added and the term `examples` was renamed to `addons` (see #24887).

Reply:
I corrected